app/assets/stylesheets/common/base/topic-post.scss

```diff
@@ -987,23 +987,30 @@ pre {
   }
 }
 
-.copy-codeblocks {
+.codeblock-buttons {
   display: block;
   position: relative;
   overflow: visible;
 
-  .copy-cmd {
+  button:nth-child(1n) {
```

Comment:
Can we target the buttons by class instead of `nth-child`? Would be easier to read and more robust.

Reply:
Hmm yes I think so, I was thinking that I wouldn't be able to because there are sometimes 1 sometimes 2 buttons, but if one of them is always present (the copy one is) then it should be OK.

Reply:
On second thought, might be good to add a wrapper element which you can position absolutely. Then it doesn't matter how many buttons are inside, or what size the buttons are (some themes might have unusually large buttons, for example).

Reply:
Yep makes sense :+1: